dashboard/OWNERS

```diff
@@ -0,0 +1,14 @@
+approvers:
+  - jlewi
```

Comment:
Please remove me. I'm already a root approver and I don't have special knowledge of the Dashboard.

Also do you want to be listed as a root approver
https://github.com/kubeflow/tf-operator/blob/master/OWNERS#L13

I know you have a different role but it might make more sense to have you focus reviewing the UI
and then also kubeflow/examples